Dockerfile

```diff
@@ -0,0 +1,22 @@
+# syntax=docker/dockerfile:1
+FROM python:3.9.2-slim AS development
+
+WORKDIR /src
+
+RUN pip install pymysql
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY . .
+
+WORKDIR /src/app
+CMD python app.py db upgrade && \
+    python app.py runserver -h 0.0.0.0 -p 8000
+
+FROM development AS production
+
+RUN pip install waitress
+
+CMD python app.py db upgrade && \
+    python waitress_wsgi.py
```

Comment:
For production, when using waitress, gunicorn or similar wsgi servers that only support http1.0 and 1.1, a reverse proxy like nginx is usefull to add http2.0 support. This can maybe be added in the compose file?

Reply:
Our outer nginx server (on pratchett itself) does http2

Reply:
I also think HTTP 2.0 shouldn't necessarily be provided by the docker image?